web/Dockerfile

```diff
@@ -13,4 +13,4 @@ COPY rootfs/ /
 
 EXPOSE 80 443
 
-VOLUME ["/config", "/etc/letsencrypt"]
+VOLUME ["/config", "/etc/letsencrypt", "/usr/share/jitsi-meet/transcripts"]
```

Comment:
why do we need this new volume?

Reply:
This volume is used inside the `docker-compose.yml` but not documented in the `Dockerfile`. I will create a new small PR for this.